src/js/helpers.js

```diff
@@ -242,11 +242,6 @@ export function filterInactivePageData(inactivePages, activePages, form) {
 }
 
 export function stringifyFormReplacer(key, value) {
-  // an object with country is an address
```

Comment:
Vets.gov will need to add back the special cases here if they need them.

Reply:
Looking at this now, I think I realize why this was added: for our address fields we default the country to `USA`. If the user is able to skip over entering the address (i.e., it's not required), the `country` field will be submitted with the form without any of the other address fields, which doesn't make a lot of sense. So in those cases, we just remove those fields.

I know we removed the address definition from our codebase that would include the logic to default the country, so this change doesn't break anything within USFS, but certainly the Vets.gov team will want to to add this special case back in. Is the only way for them to add this case back in to replace the entire `transformForSubmit` function? Or at least the `stringifyFormReplacer` part that is passed to `transformForSubmit`?